apps/coscrad-frontend/src/store/slices/resources/shared/selectors/get-select-all-for-resource-type.ts

```diff
@@ -0,0 +1,37 @@
+import { IBaseViewModel, IIndexQueryResult, ResourceType } from '@coscrad/api-interfaces';
+import { RootState } from '../../../..';
+import { ILoadable } from '../../../interfaces/loadable.interface';
+import { selectLoadableBooks } from '../../books/selectors';
+import { selectLoadablePhotographs } from '../../photographs';
+import { selectLoadableSongs } from '../../songs/selectors';
+import { selectLoadableSpatialFeatures } from '../../spatial-features/selectors';
+import { selectLoadableTerms } from '../../terms/selectors';
+import { selectLoadableTranscribedAudioItems } from '../../transcribed-audio/selectors';
+import { selectLoadableVocabularyLists } from '../../vocabulary-lists/selectors';
+
+const loookupTable: Record<
+    ResourceType,
+    (state: RootState) => ILoadable<IIndexQueryResult<IBaseViewModel>>
+> = {
+    [ResourceType.bibliographicReference]: selectLoadableBooks,
+    [ResourceType.book]: selectLoadableBooks,
+    [ResourceType.mediaItem]: selectLoadableBooks,
+    [ResourceType.photograph]: selectLoadablePhotographs,
+    [ResourceType.song]: selectLoadableSongs,
+    [ResourceType.spatialFeature]: selectLoadableSpatialFeatures,
+    [ResourceType.term]: selectLoadableTerms,
+    [ResourceType.transcribedAudio]: selectLoadableTranscribedAudioItems,
+    [ResourceType.vocabularyList]: selectLoadableVocabularyLists,
+};
+
+export const getSelectAllByResourceType = (resourceType: ResourceType) => {
```

Comment:
Double check which, if any, of these lookup table utils are used. Try to minimize it to the smallest necessary. We don't want several possible ways of doing the same thing (e.g. build the selector use the generic hook builder or build the specific hook)

Reply:
removed this one, too